src/main.rs

```diff
@@ -138,7 +138,13 @@ where
             move || {
                 while running.load(Ordering::Acquire) {
                     let render_start_time = Instant::now();
-                    let utilization = { network_utilization.lock().unwrap().clone_and_reset() };
+                    let mut utilization = { network_utilization.lock().unwrap().clone_and_reset() };
+                    if let Some(ref ex) = opts.excluded_ipv4 {
+                        utilization.remove_ip(ex)
+                    }
+                    if let Some(ref ex) = opts.excluded_ipv4_port {
+                        utilization.remove_ip_port(ex)
+                    }
```

Comment:
I don't think it makes sense to do this here. IMO there are two ways to achieve this logic and this is neither:

1. Apply the filtering in the sniffer threads, so that the filtered traffic is never accounted.
  - This is actually **not** my preferred option. Sniffers are not "hostname-aware", and therefore cannot apply hostname filters if we want to support those.
2. Apply the filtering while rendering. But then this logic belongs in `UIState::update` instead.